examples/basics/src/assets/astro.svg

```diff
@@ -0,0 +1 @@
9549 8.0498-1.9549l3.0389-9.3906c.1138-.4542.446-.7628.821-.7628.375 0 .7072.3086.821.7628l3.0388 9.3906c4.7735 0 8.0499 1.9549 8.0499 1.9549s-6.8272-18.5703-6.8405-18.6076c-.196-.549-.5267-.9026-.9727-.9026H7.8319c-.446 0-.7634.3536-.9727.9026C6.8444 11.739.018 30.31.018 30.31ZM42.3943 24.3365c0 1.6456-2.0499 2.6284-4.8883 2.6284-1.8474 0-2.5006-.4571-2.5006-1.417 0-1.0057.8109-1.4856 2.6581-1.4856 1.6672 0 3.0862.0228 4.7308.2285v.0457Zm.0226-2.0341c-1.0137-.2285-2.5457-.3656-4.3704-.3656-5.3164 0-7.8168 1.257-7.8168 4.1824 0 3.0398 1.712 4.2054 5.6767 4.2054 3.3567 0 5.6317-.8456 6.4654-2.9255h.1349c-.0225.5029-.0447 1.0057-.0447 1.3942 0 1.0742.1799 1.1656 1.0584 1.1656h4.145c-.2253-.6399-.3602-2.4455-.3602-3.9997 0-1.6684.0673-2.9254.0673-4.6167 0-3.4511-2.0722-5.6452-8.5601-5.6452-2.7933 0-5.902.4799-8.2672 1.1884.225.9371.5405 2.8341.6983 4.0683 2.0499-.96 4.9559-1.3713 7.2086-1.3713 3.1085 0 3.9648.7085 3.9648 2.1483v.5714ZM53.7899 25.297c-.563.0686-1.3292.0686-2.1176.0686-.8334 0-1.5993-.0228-2.1175-.0914 0 .1828-.0226.3885-.0226.5714 0 2.8569 1.87 4.5252 8.4478 4.5252 6.1946 0 8.1995-1.6455 8.1995-4.5481 0-2.7426-1.3288-4.0911-7.2083-4.3882-4.5732-.2057-4.9785-.7085-4.9785-1.2799 0-.6628.5856-1.0056 3.6493-1.0056 3.1763 0 4.0323.4342 4.0323 1.3485v.2056c.4507-.0228 1.2616-.0457 2.095-.0457.7884 0 1.6444.0229 2.1401.0686 0-.2057.0226-.3885.0226-.5485 0-3.3598-2.7936-4.4568-8.1999-4.4568-6.0823 0-8.1323 1.4856-8.1323 4.3882 0 2.6055 1.6447 4.2282 7.4791 4.4796 4.3025.1372 4.7757.6171 4.7757 1.2799 0 .7085-.6983 1.0285-3.7169 1.0285-3.4693 0-4.3478-.48-4.3478-1.4627v-.1372ZM73.6147 13.252c-1.6443 1.5313-4.5953 3.0626-6.2397 3.474.0225.8456.0225 2.3997.0225 3.2454l1.5092.0228c-.0225 1.6228-.0451 3.5883-.0451 4.8911 0 3.0397 1.5993 5.3252 6.5778 5.3252 2.095 0 3.4918-.2285 5.2263-.5942-.1803-1.1199-.3831-2.8341-.4507-4.1368-1.0359.3428-2.3426.5257-3.7844.5257-2.0049 0-2.8159-.5486-2.8159-2.1256 0-1.3713 0-2.6512.0226-3.8396 2.5682.0228 5.1362.0685 6.6453.1142-.0225-1.1884.0226-2.9026.0902-4.0453-2.1849.0457-4.6405.0685-6.6679.0685.0225-1.0056.045-1.9655.0676-2.9254h-.1578ZM87.0691 19.2859a767.325 767.325 0 0 1 .0676-3.1769h-4.5281c.0676 1.9656.0676 3.9768.0676 6.9251 0 2.9483-.0225 4.9825-.0676 6.9251h5.1813c-.0901-1.3713-.1127-3.6796-.1127-5.6452 0-3.1083 1.2616-3.9996 4.1225-3.9996 1.3292 0 2.2753.1599 3.1088.4571.0225-1.1657.2478-3.4283.3827-4.4339-.856-.2514-1.8021-.4114-2.951-.4114-2.4553-.0229-4.2574.9827-5.0908 3.3825l-.1803-.0228ZM109.624 22.9426c0 2.4913-1.802 3.6569-4.64 3.6569-2.816 0-4.618-1.0971-4.618-3.6569 0-2.5597 1.825-3.5197 4.618-3.5197 2.816 0 4.64 1.0285 4.64 3.5197Zm4.708-.1142c0-4.9596-3.874-7.1765-9.348-7.1765-5.4965 0-9.236 2.2169-9.236 7.1765 0 4.9367 3.4916 7.588 9.213 7.588 5.767 0 9.371-2.6513 9.371-7.588Z"/><defs><linearGradient id="a" x1="6.3277" x2="19.4266" y1="40.7999" y2="34.6018" gradientUnits="userSpaceOnUse"><stop stop-color="#D83333"/><stop offset="1" stop-color="#F041FF"/></linearGradient></defs></svg>
```

Comment:
You mentioned this being huge, so here it a bit smaller:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" width="115" height="48"><path fill="#17191E" d="M7.77 36.35C6.4 35.11 6 32.51 6.57 30.62c.99 1.2 2.35 1.57 3.75 1.78 2.18.33 4.31.2 6.33-.78.23-.12.44-.27.7-.42.18.55.23 1.1.17 1.67a4.56 4.56 0 0 1-1.94 3.23c-.43.32-.9.61-1.34.91-1.38.94-1.76 2.03-1.24 3.62l.05.17a3.63 3.63 0 0 1-1.6-1.38 3.87 3.87 0 0 1-.63-2.1c0-.37 0-.74-.05-1.1-.13-.9-.55-1.3-1.33-1.32a1.56 1.56 0 0 0-1.63 1.26c0 .06-.03.12-.05.2Z"/><path fill="url(#a)" d="M7.77 36.35C6.4 35.11 6 32.51 6.57 30.62c.99 1.2 2.35 1.57 3.75 1.78 2.18.33 4.31.2 6.33-.78.23-.12.44-.27.7-.42.18.55.23 1.1.17 1.67a4.56 4.56 0 0 1-1.94 3.23c-.43.32-.9.61-1.34.91-1.38.94-1.76 2.03-1.24 3.62l.05.17a3.63 3.63 0 0 1-1.6-1.38 3.87 3.87 0 0 1-.63-2.1c0-.37 0-.74-.05-1.1-.13-.9-.55-1.3-1.33-1.32a1.56 1.56 0 0 0-1.63 1.26c0 .06-.03.12-.05.2Z"/><path fill="#17191E" d="M.02 30.31s4.02-1.95 8.05-1.95l3.04-9.4c.11-.45.44-.76.82-.76.37 0 .7.31.82.76l3.04 9.4c4.77 0 8.05 1.95 8.05 1.95L17 11.71c-.2-.56-.53-.91-.98-.91H7.83c-.44 0-.76.35-.97.9L.02 30.31Zm42.37-5.97c0 1.64-2.05 2.62-4.88 2.62-1.85 0-2.5-.45-2.5-1.41 0-1 .8-1.49 2.65-1.49 1.67 0 3.09.03 4.73.23v.05Zm.03-2.04a21.37 21.37 0 0 0-4.37-.36c-5.32 0-7.82 1.25-7.82 4.18 0 3.04 1.71 4.2 5.68 4.2 3.35 0 5.63-.84 6.46-2.92h.14c-.03.5-.05 1-.05 1.4 0 1.07.18 1.16 1.06 1.16h4.15a16.9 16.9 0 0 1-.36-4c0-1.67.06-2.93.06-4.62 0-3.45-2.07-5.64-8.56-5.64-2.8 0-5.9.48-8.26 1.19.22.93.54 2.83.7 4.06 2.04-.96 4.95-1.37 7.2-1.37 3.11 0 3.97.71 3.97 2.15v.57Zm11.37 3c-.56.07-1.33.07-2.12.07-.83 0-1.6-.03-2.12-.1l-.02.58c0 2.85 1.87 4.52 8.45 4.52 6.2 0 8.2-1.64 8.2-4.55 0-2.74-1.33-4.09-7.2-4.39-4.58-.2-4.99-.7-4.99-1.28 0-.66.59-1 3.65-1 3.18 0 4.03.43 4.03 1.35v.2a46.13 46.13 0 0 1 4.24.03l.02-.55c0-3.36-2.8-4.46-8.2-4.46-6.08 0-8.13 1.49-8.13 4.39 0 2.6 1.64 4.23 7.48 4.48 4.3.14 4.77.62 4.77 1.28 0 .7-.7 1.03-3.71 1.03-3.47 0-4.35-.48-4.35-1.47v-.13Zm19.82-12.05a17.5 17.5 0 0 1-6.24 3.48c.03.84.03 2.4.03 3.24l1.5.02c-.02 1.63-.04 3.6-.04 4.9 0 3.04 1.6 5.32 6.58 5.32 2.1 0 3.5-.23 5.23-.6a43.77 43.77 0 0 1-.46-4.13c-1.03.34-2.34.53-3.78.53-2 0-2.82-.55-2.82-2.13 0-1.37 0-2.65.03-3.84 2.57.02 5.13.07 6.64.11-.02-1.18.03-2.9.1-4.04-2.2.04-4.65.07-6.68.07l.07-2.93h-.16Zm13.46 6.04a767.33 767.33 0 0 1 .07-3.18H82.6c.07 1.96.07 3.98.07 6.92 0 2.95-.03 4.99-.07 6.93h5.18c-.09-1.37-.11-3.68-.11-5.65 0-3.1 1.26-4 4.12-4 1.33 0 2.28.16 3.1.46.03-1.16.26-3.43.4-4.43-.86-.25-1.81-.41-2.96-.41-2.46-.03-4.26.98-5.1 3.38l-.17-.02Zm22.55 3.65c0 2.5-1.8 3.66-4.64 3.66-2.81 0-4.61-1.1-4.61-3.66s1.82-3.52 4.61-3.52c2.82 0 4.64 1.03 4.64 3.52Zm4.71-.11c0-4.96-3.87-7.18-9.35-7.18-5.5 0-9.23 2.22-9.23 7.18 0 4.94 3.49 7.59 9.21 7.59 5.77 0 9.37-2.65 9.37-7.6Z"/><defs><linearGradient id="a" x1="6.33" x2="19.43" y1="40.8" y2="34.6" gradientUnits="userSpaceOnUse"><stop stop-color="#D83333"/><stop offset="1" stop-color="#F041FF"/></linearGradient></defs></svg>
```

Reply:
Ah, I just realised that the version in the PR isn't the SVOMG version that I tried at first (It was inlined in the file), good catch